src/tokens/Assets.tsx

```diff
@@ -601,6 +602,9 @@ const styles = StyleSheet.create({
     color: Colors.gray3,
```

Comment:
can't comment on the exact line but the font should be `bodySmall` per the design [here](https://www.figma.com/file/OE6RFzqXAMqU9ygHtNYyof/Assets-Page-(Multichain)?node-id=2307%3A16037&mode=dev) and not `labelMedium`